src/quo/components/banners/alert_banner/view.cljs

```diff
@@ -11,12 +11,12 @@
             [schema.core :as schema]))
 
 (defn- view-internal
-  [{:keys [action? text button-text on-button-press]}]
+  [{:keys [action? text button-text height on-button-press]}]
```

Comment:
why not just pass a map of container-style like all other quo components?
This creates a consistent api and devs kind of know what to expect then.

Reply:
Makes sense, just updated the implementation!